Remotion/Data/DomainObjects.UnitTests/Queries/QueryFactoryTest.cs

```diff
@@ -43,7 +43,7 @@ public override void TearDown ()
     [Test]
     public void CreateQuery_FromDefinition ()
     {
-      var definition = new QueryDefinition("Test", TestDomainStorageProviderDefinition, "y", QueryType.CollectionReadWrite, typeof(OrderCollection));
+      var definition = new QueryDefinition("Test", TestDomainStorageProviderDefinition, "y", QueryType.CollectionReadWrite, QueryStatementType.Text, typeof(OrderCollection));
```

Comment:
please add assertion and one of the two tests touched in this file should use storedProc instead of Text so we demonstrate that it is actually passed through instead of hardcoded.